src/status_im/mailserver/core.cljs

```diff
@@ -182,10 +182,33 @@
     (when (= :connected mailserver-state)
       mailserver)))
 
+(fx/defn update-nil-synced-to
```

Comment:
I think in theory you can just dispatch:
```
(re-frame/dispatch [:sanitize-messages-and-process-response response])
```

below, so you don't have to write custom logic to handle it
(it would be on line 210, so `:dispatch [:sanitize.....]`)

Reply:
sure, thanks

Reply:
done